### ISSUE TITLE
Clipboard How images become png format

### ISSUE THREAD

**Issue by mofeiwo** [electron/electron#14647]
* Output of `node_modules/.bin/electron --version`:2.0.8
* Operating System (Platform and Version):MacOS

### Expected behavior
![image](https://user-images.githubusercontent.com/10068397/45607960-78ccc280-ba82-11e8-950d-1945bebddac8.png)

### Actual behavior
![image](https://user-images.githubusercontent.com/10068397/45607833-94839900-ba81-11e8-813b-544f5c4958e3.png)

```
ipcMain.on("copy",(event, arg) =>{
  const imagePath = "1.png"
  const image = nativeImage.createFromPath(imagePath)
  const { width, height } = image.getSize()
  clipboard.writeImage(nativeImage.createFromBuffer(image.toPNG()),'png')
  console.log("复制成功",width,height,clipboard.availableFormats())
})
```

**Comment by ckerr** [comment]
![tiger](https://user-images.githubusercontent.com/70381/46225896-45125800-c320-11e8-977f-8b713c368e16.png)

I'm not able to reproduce this in 2.0.8 with the attached image and the following snippet:

```js
app.on('ready', () => {
  const imagePath = '/tmp/tiger.png'
  const image = nativeImage.createFromPath(imagePath)
  const { width, height } = image.getSize()
  clipboard.writeImage(nativeImage.createFromBuffer(image.toPNG()),'png')
  console.log("复制成功",width,height,clipboard.availableFormats())
})
```

Gives the following output:

```text
Electron v2.0.8 started.
1:14:01 PM复制成功 640 640 [ 'image/png' ]
```

Tested on Ubuntu 18.04.1 in Electron Fiddle using Electron versions `2.0.8` and `3.0.2`

**Comment by ckerr** [comment]
Would it be possible for you to fork [electron-quick-start](https://github.com/electron/electron-quick-start) for a small app that reproduces the issue by itself?

Standalone test cases make fixing issues go more smoothly: it ensure everyone's looking at the same issue, it removes all unneccessary variables from the equation, and it can also provide the basis for automated regression tests.

I'm adding the `blocked/needs-repro` label for this reason. After you've responded, please link to the test case and @ me in a followup comment.

Thanks in advance! Your help is appreciated.

**Comment by mofeiwo** [comment]
@ckerr 
Open Clipboard in MacOS. In the lower left corner, you will find TIFF format. 
I hope PNG format on the clipboard.

**Comment by codebytere** [comment]
Closing as not reproducible.

**Comment by giddens9527** [comment]
Hi, I think this issue is valid and I can add some reproduce steps for this issue.
First, this issue only happens on Macos (at least on mine, Mojave 10.14.5).
What I did:
1. generate a valid data url for a png image.
2. const image = nativeImage.createFromDataURL(dataUrl);
3. clipboard.writeImage(image);
4. log clipboard.availableFormats() and it returns [ 'image/png' ].
5. View in Clipboard viewer app in mac and it shows copied file is .tiff format.

Later I tried with nativeImage.createFromPath with a .png file for step 2, and it also returned a .tiff file in clipboard viewer.
And the issue with .tiff file is the large size. (582 kb for png compared to 16 mb for tiff).
